Dockerfile

```diff
@@ -73,15 +76,24 @@ ENV DEBIAN_DISABLE_RUBYGEMS_INTEGRATION=true
 ENV BUNDLE_PATH=".bundle" \
     BUNDLE_BIN=".bundle/bin"
 ENV PATH="$BUNDLE_BIN:$PATH:$BUNDLE_PATH/bin"
-RUN apt-add-repository ppa:brightbox/ruby-ng \
-  && apt-get update \
-  && apt-get install -y --no-install-recommends ruby2.7 ruby2.7-dev \
```

Comment:
I noticed we used to install `ruby-dev` which is a package with "header files for compiling extension modules for Ruby".

AFAIK RVM installs ruby as source for the machine so this might not be a concern?

We might also want to check if this change makes the Docker build take significantly longer in case we are actually building from source instead of pulling a binary

Reply:
> I noticed we used to install ruby-dev which is a package with "header files for compiling extension modules for Ruby".
>
> AFAIK RVM installs ruby as source for the machine so this might not be a concern?

The first commit (bcf435f), I messed up the `PATH` and ended up running whatever system Ruby was available. And CI failed when attempting to install native extensions because of those missing headers.

The next commit (b591c76) fixed the `PATH` and everything worked as expected. So I think we should be arlight.

> We might also want to check if this change makes the Docker build take significantly longer in case we are actually building from source instead of pulling a binary

That shouldn't be as much of a concern on subsequent builds if the layer is cached. But yeah, it'll be interesting to see what difference that makes.